docs/install.rst

```diff
@@ -76,22 +76,21 @@ Finally, install ``stingray`` itself::
 Test Suite
 ----------
 
-Please be sure to run the test suite before you use the package, and please report anything
-you think might be bugs on our GitHub `Issues page <https://github.com/StingraySoftware/stingray/issues>`_.
+Stingray uses `py.test <https://pytest.org>`_ and `tox
+<https://tox.readthedocs.io>`_ for testing. To run the tests, try::
 
-Stingray uses `py.test <https://doc.pytest.org/en/latest/>`_ for testing. To run the tests, go into
-the ``stingray`` root directory and execute ::
+   $ tox -e test
 
-    $ python setup.py test
+You may need to install tox first::
+
+   $ pip install tox
 
 If you have installed Stingray via pip or conda, the source directory might
 not be easily accessible. Once installed, you can also run the tests using::
 
    $ python -c 'import stingray; stingray.test()'
 
-or from within a python interpreter:
-
-.. doctest-skip::
+or from within a python interpreter::
```

Comment:
`doctest-skip` is important here, or tests will be run twice! :)

Reply:
Ah I see, let me fix this.

Reply:
Thanks, I have fixed this part for both docs/install.rst and README.rst.